clean_code/Chapter 5/Formatting.md

```diff
@@ -0,0 +1,115 @@
+# 5. Formatting
+
+- You should take care that your code is nicely formatted. You should choose a set of
+  simple rules that govern the format of your code, and then you should consistently apply
+  those rules.
+
+## The Purpose of Formatting
+
+- `communication`: communication is the professional developer’s first order of business.
+- Your style and discipline survives, even though your code does not
```

Comment:
survives -> `survive`

Reply:
Copy&Pasted from book actually.. but you're right!